File: SWEA/D2/1976. 시각 덧셈/시각 덧셈.py
T = int(input())
for t in range(1, T + 1):
    h1, m1, h2, m2 = map(int, input().split())
    
    total_m = m1 + m2
    add_h = total_m // 60
    final_m = total_m % 60
    
    final_h = (h1 + h2 + add_h) % 12
    if final_h == 0:
        final_h = 12
        
    print(f"#{t} {final_h} {final_m}")

File: SWEA/D2/1976. 시각 덧셈/시각 덧셈.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        for (int t = 1; t <= T; t++) {
            int h1 = sc.nextInt();
            int m1 = sc.nextInt();
            int h2 = sc.nextInt();
            int m2 = sc.nextInt();
            
            int totalM = m1 + m2;
            int addH = totalM / 60;
            int finalM = totalM % 60;
            
            int finalH = (h1 + h2 + addH) % 12;
            if (finalH == 0) {
                finalH = 12;
            }
            
            System.out.println("#" + t + " " + finalH + " " + finalM);
        }
        sc.close();
    }
}